docs/usage/schema/vocabs.html

```diff
@@ -178,7 +178,7 @@ <h4 id="the-validation-context">The validation context</h4>
 <li><code>EvaluatedPropertyNames</code> - This is a list of property names that have been validated by either your keyword or by subschemas.  You will need to add any properties your keyword processes as well as any properties in the contexts you send to subschemas to this list.</li>
 <li><code>LocallyEvaluatedPropertyNames</code> - Similar to <code>EvaluatedPropertyNames</code> except that it only tracks those properties which have been evaluated by keywords at the current tier of the schema.</li>
 <li><code>LastEvaluatedIndex</code> - Indicates the last evaluated index in an array.</li>
-<li><code>LocakTierLastEvaluatedIndex</code> - Indicates the last evaluated index in an array by keywords at the current tier of the schema.</li>
+<li><code>LocalTierLastEvaluatedIndex</code> - Indicates the last evaluated index in an array by keywords at the current tier of the schema.</li>
```

Comment:
This one's not necessary.  The /docs folder gets regenerated as part of the build.